### PR TITLE
add documentation for styling with styled-components

### PR DESCRIPTION
**Pull request by WickedAlice**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [ ] refactoring / style
- [ ] build / chore
- [x] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [x] Yes, I've updated the documentation
- [ ] Not relevant

**Summary**

Add example for styling with styled-components

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

File: examples/Styling/Readme.md [hunk 1]
```diff
@@ -39,3 +43,39 @@ const rejectStyle = {
...
+
+const getColor = (props) => {
+    if (props.isDragActive) {    
+        return '#6c6';
+    } 
+    if (props.isDragReject) {
+        return '#c66';
+    }
+    return '#666';
+}
```
Comment (okonet) [inline examples/Styling/Readme.md]: For consistency sake, could you please add a semicolon here?

**Comment by okonet** [comment]
Thanks!

**Comment by okonet** [comment]
I’ll merge it when the base branch is green.

**Comment by rkmarks** [comment]
@okonet @WickedAlice looks like tests are still failing. I re-ran the build just to be safe and make sure it wasn't something on travis-ci. Mind taking a look and see what could be causing it to fail?

**Comment by okonet** [comment]
Yep, that’s why I wrote I’m going to merge this after tests are not failing anymore :)

**Comment by rkmarks** [comment]
So comment is mostly for @WickedAlice then lol :)

**Comment by okonet** [comment]
Up

**Pull request merged**